src/mol-model/structure/model/properties/seconday-structure.ts

```diff
@@ -20,7 +20,8 @@ namespace SecondaryStructure {
     export type Element = None | Helix | Sheet
 
     export interface None {
-        kind: 'none'
+        kind: 'none',
+        flags?: SecondaryStructureType // TODO should this level of detail be added to non-defined secondary structure elements?
```

Comment:
Sounds good to me, maybe best add a seperate Turn element in addition to None (needed for non polymers) which makes it easier to supply info when available.

Reply:
09db0ab removes capabilities to store flags from None-elements. Specifically adds Bends and Turns.

Reply:
Relates to https://github.com/molstar/molstar-proto/pull/12#discussion_r268800271 - Merged Bends and Turns in a1ed3f7